particles/Demo/RestaurantsDemo.recipes

```diff
@@ -17,12 +17,13 @@ recipe RestaurantsDemo
   create as restaurants
   create #restaurant as selected
   create #event as event
-  create as descriptions
+  // ReservationForm no longer has descriptions handle connection.
```

Comment:
I'm wondering - when the test was failing because of this field, was it showing a proper error, or did you have to investigate to find it?

Reply:
It was a reasonably clear error:

Post-parse processing error caused by 'particles/Demo/RestaurantsDemo.recipes' line 25.
param 'descriptions' is not defined by 'ReservationForm'
      descriptions = descriptions
      ^^^^^^^^^^^^^^^^^^^^^^^^^^